EKF/common.h

```diff
@@ -291,6 +291,7 @@ struct parameters {
 	float flow_noise_qual_min{0.5f};	///< observation noise for optical flow LOS rate measurements when flow sensor quality is at the minimum useable (rad/sec)
 	int32_t flow_qual_min{1};		///< minimum acceptable quality integer required once fusion of data has commenced
 	int32_t flow_qual_min_init{128};	///< minimum acceptable quality integer required to commence fusion
```

Comment:
I recommend renaming these two parameter variables and update their documentation to make the use clearer here and in the code:

int32_t flow_qual_min_init_air{128};	///< minimum acceptable quality integer required to commence fusion when in air
int32_t flow_qual_min_init_ground{1};	///<minimum acceptable quality integer required to commence fusion when on ground

Reply:
ok, but i have a question whether "flow_qual_min_init" is also used in air when _control_status.flags.opt_flow is false, so renaming flow_qual_min_init_ground is not better.

Reply:
I was confused by the pull request explanation above '2. Add a variable to enable or disable the check of minimum acceptable quality integer from the flow sensor on the ground to start using optical flow ...' 

Looking at the code this value can be used both in air and on ground so ignore the comment.

Reply:
I will read the code again to see if the use of the flow_qual_min, flow_qual_min_init and flow_qual_check_on_ground parameters is still consistent with the variables names and descriptions.

Reply:
After reading the documentation and code again, it would make it clearer if the description for the flow_qual_check_on_ground parameter was modified to read:

///< set to 1 to enable, 0 to disable application of the flow_qual_min_init threshhold check when the vehicle is on ground

I have pushed this and some other documentation changes to https://github.com/priseborough/ecl/commit/de206aeff148eebfd9ad272f75fcdef6c5e56f93

Reply:
okay. thanks!